lazer/publisher_sdk/proto/state.proto

```diff
@@ -120,6 +137,8 @@ message Feed {
     optional FeedKind kind = 108;
     // [required] Current aggregated data of the feed.
     optional FeedAggregateData aggregated_data = 109;
+    // [optional] List of Market schedules for different sessions
+    repeated FeedMarketSchedule market_schedules = 110;
```

Comment:
Let's specify in the comment what happens if both `market_schedules` and `market_schedule` are present. I suggest ignoring `market_schedule` if `market_schedules` is not empty.